examples/rust/standalone/qdrant-example/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "qdrant-example"
```

Comment:
We need a `.gitignore` in this folder, at minimum containing the `target` directory. 
Should we commit or exclude the `Cargo.lock` as well?

Reply:
Added git ignore and  committed cargo.lock. Great idea!